build.gradle

```diff
@@ -1,5 +1,5 @@
 group = 'com.github.aphoh'
```

Comment:
It doesn't make a difference, but isn't this stuff you should have done in the original PR?